lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+class TicTacToe
+    # player tokens
+    X_TOKEN = :X
+    O_TOKEN = :O
+    
+    attr_reader :player1
```

Comment:
This can also be on the same line, as you may or may not already know.

```
attr_reader :foo, :bar_chart: 
```
[`OPTIONAL`]

Reply:
fixed